src/common/BaseAnnotator.ts

```diff
@@ -191,6 +193,21 @@ export default class BaseAnnotator extends EventEmitter {
         this.store.dispatch<any>(store.fetchAnnotationsAction()); // eslint-disable-line @typescript-eslint/no-explicit-any
     }
 
+    protected removeAnnotationClasses = (): void => {
+        if (!this.annotatedEl) {
+            return;
+        }
+
+        const annotatedElement = this.annotatedEl;
+        annotatedElement.classList.remove(CSS_LOADED_CLASS);
```

Comment:
We probably want to separate out this class removal since it's only done in `destroy` vs the other classes are removed on each toggle of the annotation mode